dotCMS/src/main/java/com/dotcms/publishing/manifest/CSVManifestReader.java

```diff
@@ -0,0 +1,129 @@
+package com.dotcms.publishing.manifest;
+
+import static com.dotcms.util.CollectionsUtils.list;
+import static java.util.stream.Collectors.collectingAndThen;
+import static java.util.stream.Collectors.toSet;
+
+import com.dotcms.publishing.manifest.ManifestItem.ManifestInfo;
+import com.dotcms.publishing.manifest.ManifestItem.ManifestInfoBuilder;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.util.FileUtil;
+import com.google.common.collect.ImmutableList;
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import jersey.repackaged.com.google.common.collect.ImmutableSet;
+import org.apache.commons.io.Charsets;
+import org.apache.commons.io.FileUtils;
+
+/**
+ * Util class to read a CSV Manifest file
+ */
+public class CSVManifestReader implements ManifestReader{
+
+    final Collection<CSVManifestItem> manifestItemsIncluded;
+    final Collection<CSVManifestItem> manifestItemsExcluded;
+    final Map<String, String> metaData;
+
+    final static String COMMENTED_LINE_START = "#";
+
+    public CSVManifestReader(final File csvManifestFile){
+        try {
+            final List<String> lines = FileUtils.readLines(csvManifestFile, StandardCharsets.UTF_8);
+
+            this.manifestItemsIncluded = getManifestInfos(lines, "INCLUDED");
+            this.manifestItemsExcluded = getManifestInfos(lines, "EXCLUDED");
+            this.metaData = getAllMetaData(lines);
+        } catch (IOException e) {
+            throw new IllegalArgumentException("Not valid " + csvManifestFile.getAbsolutePath(), e);
+        }
+    }
+
+    private Map<String, String> getAllMetaData(final List<String> lines) {
+        return lines.stream()
+                .filter(line -> line.startsWith(COMMENTED_LINE_START))
+                .map(line -> line.split(":"))
```

Comment:
`StringPool.COLON` can be used instead of `:`

Reply:
done https://github.com/dotCMS/core/pull/20964/commits/112eb77d7bac9805df209b40ddd5e547901f6842#diff-ab54ca7341ae746a96f55bae50d56110b7951ff7156095d8d69d69756f243e5fR55